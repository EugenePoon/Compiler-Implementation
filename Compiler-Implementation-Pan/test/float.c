int main()
{
    float a;
    a = 1.1;
    int b;
    b = 2;
    print a+b;
}