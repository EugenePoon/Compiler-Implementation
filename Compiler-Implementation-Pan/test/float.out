24 19 0 5 25 15 1 13 26 1066192077 12 13 0 1 1 0 2 12 15 -1 13 11 13 0 1 1 11 1 22 21 2